recipes/conversation_watsonxai/tjbot.sample.toml

```diff
@@ -0,0 +1,28 @@
+[Log]
+level = 'info' # change this to 'debug' if you want to see more detail in the logs
+
+[Recipe]
+projectId = '' # FILL IN WITH YOUR WATSONX.AI PROJECT ID
+serviceUrl = 'https://us-south.ml.cloud.ibm.com' # change this if your watsonx.ai instance is in a different region
+serviceVersion = '2024-05-31'
+useNeoPixelLED = false     # set to true if using a NeoPixel LED
+useCommonAnodeLED = false  # set to true if using a Common Anode LED
```

Comment:
I wonder if we should put the `[Recipe]` config block at the top to distinguish it from "base config" stuff?